setup.py

```diff
@@ -0,0 +1,11 @@
+from setuptools import setup
+
+setup(name='ponyo',
+version='0.1',
```

Comment:
You may want to add indentation on lines 4-11, since they are all arguments to `setup()` function.